app/controllers/api/pods_controller.rb

```diff
@@ -113,8 +113,17 @@ def verify_pushes_allowed!
                       'your version of CocoaPods to push this specification.'
             json_error(422, message)
           end
+
+          if Version.new(version[1]) == Version.new("1.3.0") 
```

Comment:
We only have JSON data at this point @dnkoutso (though I could turn that into a Podspec)- I think I'm okay with just removing the entire version, it's only been the prod version for a week or two.